incubator/java-openliberty/image/project/pom.xml

```diff
@@ -38,7 +38,26 @@
             <url>/mvn/repository</url>
         </repository>
     </repositories>
-
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.openliberty.features</groupId>
+                <artifactId>features-bom</artifactId>
+                <version>${version.openliberty-runtime}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+    <dependencies>
+         <dependency>
+            <groupId>org.eclipse.microprofile</groupId>
```

Comment:
If you specify the features-bom, why do you need to specify MicroProfile dependency?

Reply:
I was trying to cache the MicroProfile feature in a docker layer. I may not completely understand the mechanism here though. If I add the bom to I actually end up getting all the features?